numberformat/diff.emu

```diff
@@ -123,13 +123,15 @@
             1. Let _mxfdDefault_ be 0.
           1. Else,
             1. Let _mxfdDefault_ be 3.
+        1. <ins class="block">Let _roundingIncrement_ be ? GetNumberOption(_options_, *"roundingIncrement"*, 1, 5000, 1).</ins>
+        1. <ins class="block">If _roundingIncrement_ is not in « 1, 2, 5, 10, 20, 25, 50, 100, 200, 250, 500, 1000, 2000, 2500, 5000 », throw a *RangeError* exception.</ins>
+        1. <ins class="block">If _roundingIncrement_ is not 1, set _mxfdDefault_ to _mnfdDefault_.</ins>
```

Comment:
Thought: The other option would be to set `_mxfdDefault_` only if the `roundingIncrement` field in `_options_` is not `undefined`. But, what you have here is perfectly fine. The difference is what happens with

```javascript
new Intl.NumberFormat("en", { roundingIncrement: 1 })
```

Reply:
I was considering an alternative version of this PR doing something similar to that,  but also moving part of this logic to `SetNumberFormatDigitOptions` where we already doing a similar check for ` roundingPriority`.  but found this one more straightforward, otherwise, we split validations or move things to the same place to get more consistency.